src/pages/checkout.jsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from "react";
+
+const Checkout = ({ basketItems }) => {
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [phone, setPhone] = useState("");
+  const [street, setStreet] = useState("");
+  const [cardNumber, setCardNumber] = useState("");
```

Comment:
use formValues object as suggested before always when there are many form fields, in checkout page usally form fields are alot